packages/components/checkbox/src/CheckboxLabel.tsx

```diff
@@ -9,8 +9,8 @@ export interface CheckboxLabelProps extends LabelProps, LabelStylesProps {
   disabled?: boolean
 }
 
-export const CheckboxLabel = ({ className, disabled, ...others }: CheckboxLabelProps) => {
-  return <Label className={labelStyles({ className, disabled })} {...others} />
```

Comment:
Why are you removing the `className` here ? 🤔

Reply:
The `className` was previously provided to `Label` as it was wrapping the whole component. Now it has been changed, so the prop will remain on the main wrapper, not here anymore.

Reply:
Ok I see. However, as an agreement we said that all components should extend the `className`

Reply:
Oh, didn't know. But does this agreement concern also "subcomponents"? As this is not directy exposed and do not have any usage of this prop, it doesn't make really sense to me... don't you think? :thinking:

Reply:
any sub-component exposed (exported) must follow the same agreements